etna/datasets/tsdataset.py

```diff
@@ -533,14 +533,16 @@ def plot(
             ax[i].grid()
 
     @staticmethod
-    def to_flatten(df: pd.DataFrame) -> pd.DataFrame:
+    def to_flatten(df: pd.DataFrame, columns: Optional[List[str]] = None) -> pd.DataFrame:
         """Return pandas DataFrame with flatten index.
 
         Parameters
         ----------
         df:
             DataFrame in ETNA format.
-
+        columns:
+            List of columns to return
```

Comment:
Don't we want to check that columns are present? Or we are content with error from pandas itself?

Reply:
Actually I do not get an error on pandas 1.3.5, anyway error from pandas won't be ambiguous, it will be clear that there is no such column in the dataset